src/user/views/user_views.py

```diff
@@ -72,6 +78,41 @@ def get_serializer_class(self):
     def get_serializer_context(self):
         return {"get_subscribed": True, "get_balance": True, "user": self.request.user}
 
+    @action(detail=False, methods=["GET"], permission_classes=[IsAuthenticated])
+    def get_referred_users(self, request):
+        invited = User.objects.filter(invited_by=request.user).annotate(
+            rsc_earned=Sum(
+                "reputation_handed_out__amount",
+                filter=(
+                    Q(
+                        reputation_handed_out__distribution_type="REFERRAL_REFERER_EARNINGS"
+                    )
+                    & Q(reputation_handed_out__recipient_id=request.user)
+                ),
+            ),
+            benefits_expire_on=models.ExpressionWrapper(
+                models.F("created_date")
+                + timedelta(
+                    days=REFERRAL_PROGRAM["ELIGIBLE_TIME_PERIOD_IN_MONTHS"] * 30
```

Comment:
@le-sun I can't seem to be able to do this in months. Like `months=6`. Tried using relativedelta but doesn't work here. any ideas?